File: client/src/Components/Review.js
import React from 'react';
import './Review.css';
import { Rate } from 'antd';

const Review = ({ info }) => {
  return (
    <div className="review">
      <h3>Review by {info.author}</h3>
      <Rate allowHalf defaultValue={info.rating} disabled className="rating" />
      <div className="desc">"{info.description}"</div>
    </div>
  );
};

export default Review;

File: RestAPI/Domain/Models/review.cs
namespace RestAPI.Domain.Models
{
    public class Review
    {
        public int ReviewId { get; set; }

        public int MovieId { get; set; }

        public string Author { get; set; }

        public decimal Rating { get; set; }

        public string Description { get; set; }
    }
}
